### PR TITLE
try my best to translate in french

### PR DESCRIPTION
**Pull request by papiche**

### CODE REVIEW

**Comment by oflg** [comment]
@papiche Thanks! 

That's great. I'll add it to the Tidme repository later.

File: tiddlers/Boudoir.tid [hunk 1]
```diff
@@ -0,0 +1,12 @@
+created: 20141029161248510
+filter: Lire un article Mémoriser des notes Installation et utilisation Apprendre une langue étrangère
+modified: 20220515085335668
+tags: $:/tags/TiddlerList
+title: Boudoir
```
Comment (oflg) [inline tiddlers/Boudoir.tid]: Same with [this comment](https://github.com/oflg/Tidme-manual-zh-Hans/pull/1/commits/6985a0138be32881079cb0c5fc0208fcb48b277d#r1388914167) .

**Review by oflg — COMMENTED** [review]
fix: missing link

**Comment by papiche** [comment]
I am not good at git. I am trying to try many Tiddlywiki use case
for Umap TW project https://talk.tiddlywiki.org/t/uplanet-a-planet-full-of-tws/8193/13?u=papiche ?

**Pull request closed without merging**